client/a8c-for-agencies/components/items-dashboard/items-dataviews/interfaces.ts

```diff
@@ -0,0 +1,71 @@
+import { ReactNode } from 'react';
+
+export interface ItemsDataViewsType< T > {
```

Comment:
`T` is the item type. This is set by the implementation, for Dotcom is `SiteExcerptData` and for A4A is `SiteData`